src/ansys/mapdl/core/misc.py

```diff
@@ -519,33 +519,27 @@ def create_temp_dir(tmpdir=None, name=None):
     """Create a new unique directory at a given temporary directory"""
     if tmpdir is None:
         tmpdir = tempfile.gettempdir()
-    elif not os.path.isdir(tmpdir):
-        os.makedirs(tmpdir)
 
-    if not name:
-        random_name = True
-        letters_ = string.ascii_lowercase.replace("n", "")
-        name = random_string(10, letters_)
-    else:
-        random_name = False
+    # Possible letters
+    letters_ = string.ascii_lowercase.replace("n", "")
```

Comment:
Why not use `[tempfile.mkdtemp](https://docs.python.org/3/library/tempfile.html#tempfile.mkdtemp)` ?
Is it for the length of the folder name ?

Reply:
That's actually a good idea. I will try to use it.